.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
       node_js:
         - 'lts/*'
     - name: "Develop UI Tests"
-      env: SRC_PATH='examples/gatsbygram' INSTALL_COMMAND='install' TEST_COMMAND='test'
+      env: SRC_PATH='examples/gatsbygram' INTEGRATION_TEST=true INSTALL_COMMAND='install' TEST_COMMAND='test'
```

Comment:
General idea here is that any "integration/e2e test" simply needs to add the `INTEGRATION_TEST=true` var to the job matrix, and it will automatically get _all_ the changed content in the PR.

Reply:
I think this is great change!